src/test/java/org/testeditor/fixture/host/HostDriverFixtureTest.java

```diff
@@ -17,28 +19,27 @@
   private final int MAX_ROWS = 24;
   private final int MAX_COLUMNS = 80;
   private final String STANDARD_WINDOW_ID = "0x0";
+  private final String S2370_PATH = System.getenv("S3270_PATH");
+  private final String hostUrl = System.getenv("HOST_URL");
+  private final int hostPort = Integer.parseInt(System.getenv("HOST_PORT"));
+  HostDriverFixture hdf;
 
-  @Test
-  public void connectionTest() {
-    String S2370_PATH = System.getenv("S3270_PATH");
-    String hostUrl = System.getenv("HOST_URL");
-    int hostPort = Integer.parseInt(System.getenv("HOST_PORT"));
-    HostDriverFixture hdf = new HostDriverFixture();
+  @Before
+  public void init() {
+    hdf = new HostDriverFixture();
     hdf.connect(S2370_PATH, hostUrl, hostPort);
```

Comment:
why not have somthing like assumeTrue(hostDriverFixture.connected()); within the tests and check availibility of env parameters before actually attempting the connect here?